Twig/Extension/FormatterMediaExtension.php

```diff
@@ -76,14 +76,6 @@ public function getTwigExtension()
     }
 
     /**
-     * {@inheritdoc}
-     */
-    public function getName()
```

Comment:
This has been released and breaks things , it seems. Can you follow up on this? See #1215